.github/buildomat/jobs/omicron-common.sh

```diff
@@ -21,5 +21,6 @@ cd /tmp
 cargo new --lib test-project
 cd test-project
 cargo add omicron-common --path /work/oxidecomputer/omicron/common
+cp /work/oxidecomputer/omicron/Cargo.lock Cargo.lock
```

Comment:
I guess what happens is that when Cargo runs it will reduce the set of deps from `Cargo.lock` but not bump versions. Could you add a comment explaining this?

Reply:
yeah that's exactly my expectation, though i don't think it matters much if cargo ignores or prunes extra deps. but yeah a comment in the script is probably a bit nicer than a commit message :grin:

done w/ https://github.com/oxidecomputer/omicron/pull/6692/commits/278ca2e40b1e643367f9086097500770325319ef

Reply:
Thanks!